### DISCUSSION TITLE
Where do I report a bug?

### DISCUSSION THREAD

**Discussion by peppermintbunny** [supabase/supabase#13919] [Questions] [answered]
I've come across a bug but I don't know where / how to report this.  Please advise.  Many thanks

[If one uses the new "Definition" option to display a table's structure in SQL there is an error in the SQL. Array datatypes are not converted correctly.  Their datatype is omitted.  So, for example a table field named PURCHASES which is defined in the Table Editor as being of  TEXT ARRAY is converted to SQL merely as " PURCHASES ARRAY...."  The datatype is not reported.]

**Comment by j4w8n** [comment, marked as answer]
You can create an issue in this repo. If they determine it needs to go in another supabase repo, someone will move it.

**Reply by peppermintbunny** [reply]
Thanks!
